Hadrons/Modules/MContraction/Meson.hpp

```diff
@@ -153,14 +155,20 @@ template <typename FImpl1, typename FImpl2>
 std::vector<std::string> TMeson<FImpl1, FImpl2>::getOutput(void)
 {
     std::vector<std::string> output = {};
+    if (!par().output.length())
```

Comment:
The logic is ok but somehow different compared to the rest of Hadrons. I would prefer the output to be saved in the environement in any case (it is very small), and that it is saved on disk if `par().output` is not empty.